src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -181,6 +185,25 @@ const fetchUserData = async () => {
 	});
 };
 
+const isFirstLoan = computed(() => {
+	return (isGuest.value || !userData.value?.my?.loans?.totalCount) && basketCount.value <= 1;
```

Comment:
Do we need the basket count to be <=1? Technically each of their loans will apply towards equity, so should we show the equity badge or each entry like we do with other milestones?